support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummaryCopy.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+const cancellationCopy = {
+  title: 'Cancel at any time',
+  body: 'There is no set time on your agreement with us so you can end your subscription whenever you wish',
+};
```

Comment:
because it's used in two places, I thought it would be better to help keep it consistent. The other option I thought of was too have one component and change the styling of the component depending on where the copy was shown e.g. in the end summary or the side bar, but this seemed more complicated and unclear

Reply:
Ah, got it. 👍